src/services/services.ts

```diff
@@ -4349,7 +4349,7 @@ namespace ts {
                                         displayParts.push(keywordPart(SyntaxKind.NewKeyword));
                                         displayParts.push(spacePart());
                                     }
-                                    if (!(type.flags & TypeFlags.Anonymous)) {
+                                    if (!(type.flags & TypeFlags.Anonymous) && type.symbol) {

```

Comment:
If we allow anything with `.symbol`, anonymous function tooltips will look like `var lambdaFoo: (Anonymous function)(a: number, b: number) => number'`.